### DISCUSSION TITLE
Support for Hatch

### DISCUSSION THREAD

**Discussion by GH-maggio** [DavidVujic/python-polylith#100] [Ideas]
Any plans on porting this tool to [Hatch](https://github.com/pypa/hatch) ?

**Comment by DavidVujic** [comment]
I haven't any current plans for Hatch in specific, but plan to make the poly tool less coupled to Poetry - if all goes well I'm thinking of publishing the Poly tool also as a separate CLI. This will probably make it more compatible with Hatch. I just quickly browsed the docs for Hatch and it seems like a nice tool, looks like it even supports relative includes.

**Reply by GH-maggio** [reply]
A stand alone CLI sounds like a great idea, Ill keep an eye out for it in the future.

**Comment by DavidVujic** [comment]
Quick update: I'm working on this right now and a first step towards supporting Hatch!
https://github.com/DavidVujic/python-polylith/pull/144

**Comment by DavidVujic** [comment]
Officially supporting Hatch today!

Announcement here: https://github.com/DavidVujic/python-polylith/discussions/158
